data/kahet/beyond patir assets.txt

```diff
@@ -0,0 +1,2471 @@
+	"velocity scale" 0.
+
+effect "jump drive 2"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 3"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 4"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 5"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 6"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 7"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 8"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 9"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 10"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 11"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 12"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 13"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 14"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive 15"
+	"lifetime" 10
+	"random angle" 180
+	"velocity scale" 0.
+
+effect "jump drive red"
+	sprite "effect/jump drive red"
+		"no repeat"
+		"frame rate" 8
+		"rewind"
+	"lifetime" 8
+	"velocity scale" 3
+
+effect "jump drive circle"
+	sprite "effect/jd detainer"
+		"no repeat"
+		"frame time" 1.5
+	"lifetime" 50
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 1"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 2"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 3"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 4"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 5"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 6"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 7"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 8"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+effect "jump drive spark 9"
+	sprite "effect/repulsor impact"
+		"frame time" 6
+		"random start frame"
+	"lifetime" 15
+	"random spin" 3
+	"random angle" 180
+
+outfit "Asteroid Weapon Large 4"
```

Comment:
I like the idea behind this - a projectile that flies off and lays mines.

It probably ought to have some negative damage on the earlier projectiles so that just sticking it into the target at point-blank range doesn't immediately hit them with all 84 mines' worth of damage.

Reply:
Added negative corrosion, hull and slowing damage to each print. This makes it so only 22.9 hull damage, no corrosion and 1 slowing per mine are carried to the previous projectiles; this means hitting the very first projectile would impact nearly 2000 hull damage and 84 slowing damage, for a weapon that shoots every 130 frames